### PR TITLE
AB#895 Create initial message creation modal (+ minimalization)

### PR DESCRIPTION
**Pull request by mkidawa**

### CODE REVIEW

**Review by szymonbla — APPROVED** [review]
LGTM

File: src/redux/messageCreationModal.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { RootState } from './store';
+
+interface ModalState {
+  isOpen: boolean;
+  isHidden: boolean;
```
Comment (JKrymarys) [inline src/redux/messageCreationModal.ts:6]: `isMinimised?` or `isCollapsed? ` ? 

isHidden sounds like isOpen=false

**Review by JKrymarys — APPROVED** [review]
Approved, but please take a look at the variable I commented

File: src/redux/messageCreationModal.ts (hunk 1 above)
Comment (mkidawa) [reply src/redux/messageCreationModal.ts:6]: Definitely agree :D

**Pull request merged**